app/views/layouts/application.html.erb

```diff
@@ -96,6 +96,9 @@
               <li class="govuk-footer__inline-list-item">
                 <%= link_to t('layout.accessibility'), accessibility_path, class: 'govuk-footer__link' %>
               </li>
+              <li class="govuk-footer__inline-list-item">
```

Comment:
Let's figure something out on Monday about splitting up the layouts for candidates/providers.